Sources/WebAuthn/WebAuthn.swift

```diff
@@ -3,9 +3,19 @@ import Crypto
 import Logging
 import Foundation
 
+extension String {
+    var base64URLDecodedData: Data? {
+        var result = self.replacingOccurrences(of: "-", with: "+").replacingOccurrences(of: "_", with: "/")
+        while result.count % 4 != 0 {
+            result = result.appending("=")
+        }
+        return Data(base64Encoded: result)
+    }
+}
+
 public enum WebAuthn {
     public static func validateAssertion(_ data: AssertionCredential, challengeProvided: String, publicKey: P256.Signing.PublicKey, logger: Logger) throws {
-        guard let clientObjectData = Data(base64Encoded: data.response.clientDataJSON) else {
+        guard let clientObjectData = data.response.clientDataJSON.base64URLDecodedData else {
```

Comment:
I'll need to check the spec here. In my testing the assertion was _not_ base64 USL decoded, so it may depend on the assertion and we might need to check some data to see which path to go down

Reply:
Yep it's URL encoded, thanks!